composer/loggers/logger_hparams.py

```diff
@@ -89,7 +90,8 @@ class WandBLoggerBackendHparams(BaseLoggerBackendHparams):
     """
 
     project: Optional[str] = hp.optional(doc="wandb project name", default=None)
-    name: Optional[str] = hp.optional(doc="wandb group name", default=None)
+    group: Optional[str] = hp.optional(doc="wandb group name", default=None)
```

Comment:
If the group is not set and `rank_zero_only` is True, could you set the group to the value of `name`? This will help make it backwards compatible with the existing YAMLs, so all ranks of the same job will be auto-grouped together.